specification/storage/resource-manager/Microsoft.Storage/stable/2019-04-01/blob.json

```diff
@@ -688,6 +688,56 @@
             }
           }
         }
+      },
+      "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/Microsoft.Storage/storageAccounts/{accountName}/blobServices/default/containers/{containerName}/lease": {
```

Comment:
```suggestion
      "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Storage/storageAccounts/{accountName}/blobServices/default/containers/{containerName}/lease": {
```

Reply:
Thanks! This is fixed in latest iterations.